src/crimson/os/alienstore/thread_pool.h

```diff
@@ -15,14 +15,7 @@
 #include <seastar/core/semaphore.hh>
 #include <seastar/core/sharded.hh>
 
-#if __cplusplus > 201703L
-#include <semaphore>
-namespace crimson {
-  using std::counting_semaphore;
-}
-#else
```

Comment:
i concur with Sam, better off adding a comment explaining the rationales behind this.

Reply:
Yeah, will do.